test/functional/p2p_leak.py

```diff
@@ -64,28 +65,15 @@ def on_getblocktxn(self, message): self.bad_message(message)
     def on_blocktxn(self, message): self.bad_message(message)
 
 
-# Node that never sends a version. We'll use this to send a bunch of messages
-# anyway, and eventually get disconnected.
-class CNodeNoVersionMisbehavior(CLazyNode):
-    pass
-
-
-# Node that never sends a version. This one just sits idle and hopes to receive
-# any message (it shouldn't!)
-class CNodeNoVersionIdle(CLazyNode):
-    def __init__(self):
-        super().__init__()
-
-
-# Node that sends a version but not a verack.
-class CNodeNoVerackIdle(CLazyNode):
+# Peer that sends a version but not a verack.
+class CNoVerackIdlePeer(CLazyPeer):
     def __init__(self):
         self.version_received = False
         super().__init__()
 
     def on_verack(self, message): pass
     # When version is received, don't reply with a verack. Instead, see if the
-    # node will give us a message that it shouldn't. This is not an exhaustive
+    # peer will give us a message that it shouldn't. This is not an exhaustive
```

Comment:
This is actually waiting to see if the bitcoind node sends an unexpected messaged, so the word 'node' is correct here.

Reply:
Nice catch. Thank you!